src/chk/check.pb-c.c

```diff
@@ -0,0 +1,317 @@
+  { "CIA_READD", 4 },
+  { "CIA_TRUST_EC_DATA", 12 },
+  { "CIA_TRUST_EC_PARITY", 11 },
+  { "CIA_TRUST_LATEST", 9 },
+  { "CIA_TRUST_MAJORITY", 8 },
+  { "CIA_TRUST_MS", 5 },
+  { "CIA_TRUST_OLDEST", 10 },
+  { "CIA_TRUST_PS", 6 },
+  { "CIA_TRUST_TARGET", 7 },
+};
+const ProtobufCEnumDescriptor chk__check_inconsist_action__descriptor =
+{
+  PROTOBUF_C__ENUM_DESCRIPTOR_MAGIC,
+  "chk.CheckInconsistAction",
+  "CheckInconsistAction",
+  "Chk__CheckInconsistAction",
+  "chk",
+  13,
+  chk__check_inconsist_action__enum_values_by_number,
+  13,
+  chk__check_inconsist_action__enum_values_by_name,
+  1,
+  chk__check_inconsist_action__value_ranges,
+  NULL,NULL,NULL,NULL   /* reserved[1234] */
+};
+static const ProtobufCEnumValue chk__check_flag__enum_values_by_number[5] =
+{
+  { "CF_NONE", "CHK__CHECK_FLAG__CF_NONE", 0 },
+  { "CF_DRYRUN", "CHK__CHECK_FLAG__CF_DRYRUN", 1 },
+  { "CF_RESET", "CHK__CHECK_FLAG__CF_RESET", 2 },
+  { "CF_FAILOUT", "CHK__CHECK_FLAG__CF_FAILOUT", 4 },
+  { "CF_AUTO", "CHK__CHECK_FLAG__CF_AUTO", 8 },
+};
+static const ProtobufCIntRange chk__check_flag__value_ranges[] = {
+{0, 0},{4, 3},{8, 4},{0, 5}
+};
+static const ProtobufCEnumValueIndex chk__check_flag__enum_values_by_name[5] =
+{
+  { "CF_AUTO", 4 },
+  { "CF_DRYRUN", 1 },
+  { "CF_FAILOUT", 3 },
+  { "CF_NONE", 0 },
+  { "CF_RESET", 2 },
+};
+const ProtobufCEnumDescriptor chk__check_flag__descriptor =
+{
+  PROTOBUF_C__ENUM_DESCRIPTOR_MAGIC,
+  "chk.CheckFlag",
+  "CheckFlag",
+  "Chk__CheckFlag",
+  "chk",
+  5,
+  chk__check_flag__enum_values_by_number,
+  5,
+  chk__check_flag__enum_values_by_name,
+  3,
+  chk__check_flag__value_ranges,
+  NULL,NULL,NULL,NULL   /* reserved[1234] */
+};
+static const ProtobufCEnumValue chk__check_inst_status__enum_values_by_number[7] =
+{
+  { "CIS_INIT", "CHK__CHECK_INST_STATUS__CIS_INIT", 0 },
+  { "CIS_RUNNING", "CHK__CHECK_INST_STATUS__CIS_RUNNING", 1 },
+  { "CIS_COMPLETED", "CHK__CHECK_INST_STATUS__CIS_COMPLETED", 2 },
+  { "CIS_STOPPED", "CHK__CHECK_INST_STATUS__CIS_STOPPED", 3 },
+  { "CIS_FAILED", "CHK__CHECK_INST_STATUS__CIS_FAILED", 4 },
+  { "CIS_PAUSED", "CHK__CHECK_INST_STATUS__CIS_PAUSED", 5 },
+  { "CIS_CRASHED", "CHK__CHECK_INST_STATUS__CIS_CRASHED", 6 },
+};
+static const ProtobufCIntRange chk__check_inst_status__value_ranges[] = {
+{0, 0},{0, 7}
+};
+static const ProtobufCEnumValueIndex chk__check_inst_status__enum_values_by_name[7] =
+{
+  { "CIS_COMPLETED", 2 },
+  { "CIS_CRASHED", 6 },
+  { "CIS_FAILED", 4 },
+  { "CIS_INIT", 0 },
+  { "CIS_PAUSED", 5 },
+  { "CIS_RUNNING", 1 },
+  { "CIS_STOPPED", 3 },
+};
+const ProtobufCEnumDescriptor chk__check_inst_status__descriptor =
+{
+  PROTOBUF_C__ENUM_DESCRIPTOR_MAGIC,
+  "chk.CheckInstStatus",
+  "CheckInstStatus",
+  "Chk__CheckInstStatus",
+  "chk",
+  7,
+  chk__check_inst_status__enum_values_by_number,
+  7,
+  chk__check_inst_status__enum_values_by_name,
+  1,
+  chk__check_inst_status__value_ranges,
+  NULL,NULL,NULL,NULL   /* reserved[1234] */
```

Comment:
(style)  space required after that ',' (ctx:VxV)
(style)  space required after that ',' (ctx:VxV)
(style)  space required after that ',' (ctx:VxV)